src/ui/src/wds/WdsCollapsable.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<details
+		class="wds-collapsable"
```

Comment:
Preferred spelling is "collapsible", please change.

https://github.com/dotnet/fsharp/issues/5247